### DISCUSSION TITLE
歡迎光臨得來超速

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#567] [General]
這樣以後我就可以更快吃道麥當勞，水啦!
https://tw.stock.yahoo.com/news/%E9%A3%9F%E5%8A%9B-%E9%BA%A5%E7%95%B6%E5%8B%9E%E9%A6%96%E9%96%93%E5%8D%8A%E8%87%AA%E5%8B%95%E5%BA%97%E6%96%BC%E7%BE%8E%E5%9C%8B%E5%BE%B7%E5%B7%9E%E9%96%8B%E5%B9%95-ai%E6%9C%83%E6%94%B9%E8%AE%8A%E9%80%9F%E9%A3%9F%E6%A5%AD%E8%B5%B0%E5%90%91%E8%87%AA%E5%8B%95%E5%8C%96%E7%9A%84%E6%9C%AA%E4%BE%86%E5%97%8E-024615566.html

**Comment by awehao** [comment]
這跟外送自取好像差不多，但若是越接近的人越早備餐，是否會引想先點後到的其他顧客，這點又有失公平性，感覺值得深思

**Comment by Chenszuhong** [comment]
這感覺就還蠻不錯的 喜歡麥當勞的可以更方便

**Comment by lukiea** [comment]
要想清楚，麥當勞，到底是一種怎麼樣的存在。

若無法徹底理解麥當勞，恐怕會是人類的一大遺憾。動機，可以說是最單純的力量。

**Reply by awehao** [reply]
我自己打老師都給我眼睛了，我來幫老師給你眼睛

**Reply by chia-kai-chang** [reply]
> 我自己打老師都給我眼睛了，我來幫老師給你眼睛

我好想笑 哈哈😆

**Comment by manipulateed** [comment]
優點很明顯是這些自動化的發展，可以製作出更標準化的商品減少瑕疵或是人為疏失的造成，但就是失業的問題也很明顯，許多工讀生或是需要簡單工作來補貼家用的人，可能就此要去尋找還需要人來操作的工作了。

**Comment by larry40305** [comment]
以後就會結合自動車達成自動配送。在家點餐幾分鐘後，就會有機器人或無人機送到你家了。

**Comment by magicmicheal** [comment]
之前有看到這方面的負面報導就是民眾拒絕買自動化送餐的商品，讓企業家意識到，不能為了營利造成失業率大大上升

**Reply by RogelioKG** [reply]
我覺得這是很好的觀點。
就像經濟學家凱因斯所認為的「我們想出辦法用機器取代人類的速度，將超過找到新方法運用勞力的速度」，如果這樣半自動化的營業模式逐漸風行，在沒有任何配套措施的情況下，確實很可能會造成大規模的技術性失業。

**Comment by awkward-willy** [comment]
這樣的技術能讓趕時間的民眾更快速的得到自己想要的餐點，也能減少人力在低技術取向工作的需求，甚至能改善人工勞累，送錯餐，薪資談不攏等問題，是相當不錯的。當然另一方面而言，失業率上升、民眾體驗差距等議題也是值得探討的。

**Comment by 0212Jerry** [comment]
全自動化的麥當勞確實可以省下許多人事成本。像現在許多麥當勞都有自助點餐的機器出現，讓點餐不再需要人擠人，在疫情下也完美的解決了人與人接觸的問題

**Comment by Vincent20030314** [comment]
這項技術讓喜歡麥當勞的我更喜歡去買麥當勞了!
